src/components/dialogmote/motehistorikk/MotehistorikkPanel.tsx

```diff
@@ -129,6 +134,9 @@ export const MotehistorikkPanel = ({
       {historiskeMoter.map((mote, index) => (
         <MoteHistorikk key={index} mote={mote} />
       ))}
+      {dialogmoteunntak.map((unntak, index) => (
+        <MoteHistorikkUnntak key={index} unntak={unntak} />
+      ))}
```

Comment:
Har ikke laget noe sortering/inndeling som gjør at det er tydelig med rekkefølge på unntak vs dialogmøte-endringer, men det kan jo evt. ordnes etterhvert.

Det er ikke noe skisse på dette ennå, så har laget slik at det er så konsistent som mulig med tilsvarende opplegg for dialogmøte-endringer.